src/templates/nssfcfg.conf.j2

```diff
@@ -0,0 +1,85 @@
+configuration:
+  nrfUri: {{ nrf_url }}
+  nsiList:
+  - nsiInformationList:
+	- nrfId: {{ nrf_url }}/nnrf-nfm/v1/nf-instances
+	  nsiId: 22
+  nssfName: NSSF
+  sbi:
+	bindingIPv4: 0.0.0.0
+	port: {{ sbi_port }}
+	registerIPv4: {{ nssf_url }}
+	scheme: http
+  serviceNameList:
+  - nnssf-nsselection
+  - nnssf-nssaiavailability
+info:
+  description: NSSF initial local configuration
+  version: 1.0.0
+logger:
+  AMF:
+	ReportCaller: false
+	debugLevel: info
+  AUSF:
```

Comment:
Do we need all these loggers here? Shouldn't each NF configure it's own logger?

Reply:
I removed all the loggers.